eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/trace/Trace.java

```diff
@@ -120,7 +121,7 @@ public Span addTraceInfoToSpan(ChannelHandlerContext ctx, CloudEvent cloudEvent)
 
         //add trace info
         for (String entry : cloudEvent.getExtensionNames()) {
-            span.setAttribute(entry, cloudEvent.getExtension(entry) == null ? "" : cloudEvent.getExtension(entry).toString());
+            span.setAttribute(entry, cloudEvent.getExtension(entry) == null ? "" : Objects.requireNonNull(cloudEvent.getExtension(entry)).toString());
```

Comment:
The judgment of ternary operation ensures that 'cloudEvent. getExtension (entry)' cannot return null here. `Objects.requireNonNull()` is redundant.